overlays/nodejs/lambda/lambdaMixins.ts

```diff
@@ -0,0 +1,302 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as pulumi from "@pulumi/pulumi";
+import * as crypto from "crypto";
+
+import * as arn from "../arn";
+import * as iam from "../iam";
+import * as runtime from "./runtimes";
```

Comment:
Nit, strange sorting, I'd expect `runtime` to be next to `lambdaFunction` and `permission` since both are in the current directory.